File: SDKs/Aspose.Words_Cloud_SDK_for_CSharp/src/Com/Aspose/Words/Model/FieldNames.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;

namespace Com.Aspose.Words.Model {
  public class FieldNames {
    public List<string> Names { get; set; }

    public Link link { get; set; }

    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class FieldNames {\n");
      sb.Append("  Names: ").Append(Names).Append("\n");
      sb.Append("  link: ").Append(link).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  }

File: SDKs/Aspose.Words-Cloud-SDK-for-Java/src/main/java/com/aspose/words/model/FieldNames.java
package com.aspose.words.model;

import java.util.*;
import com.aspose.words.model.Link;
public class FieldNames {
  private List<String> Names = new ArrayList<String>();
  private Link link = null;
  /**
	 * getNames
	 * Gets List<String>
	 * @return Names
	 */
  public List<String> getNames() {
    return Names;
  }

	/**
	 * setNames
	 * Sets List<String>
	 * @param Names List<String>
	 */
  public void setNames(List<String> Names) {
    this.Names = Names;
  }

  /**
	 * getLink
	 * Gets Link
	 * @return link
	 */
  public Link getLink() {
    return link;
  }

	/**
	 * setLink
	 * Sets Link
	 * @param link Link
	 */
  public void setLink(Link link) {
    this.link = link;
  }

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class FieldNames {\n");
    sb.append("  Names: ").append(Names).append("\n");
    sb.append("  link: ").append(link).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}

